BikeRental.Contracts/Interfaces/IAnalyticsService.cs

```diff
@@ -0,0 +1,41 @@
+using BikeRental.Domain.Enums;
+using BikeRental.Contracts.Dtos;
+
+namespace BikeRental.Contracts.Interfaces
```

Comment:
file-scoped.
```suggestion
namespace BikeRental.Contracts.Interfaces;
```
Таких замечаний тоже прилично в студии.
<img width="1343" height="163" alt="image" src="https://github.com/user-attachments/assets/ba16755a-6034-4f55-b239-999200281e79" />